src/development/add-to-app/ios/add-flutter-screen.md

```diff
@@ -239,6 +357,87 @@ The `FlutterAppDelegate` performs functions such as:
   (which can only be detected in the AppDelegate) to
   Flutter for scroll-to-top behavior.
 
+### Creating a FlutterAppDelegate subclass
```

Comment:
it's very possible that they already have an app delegate. maybe also document what to do if you can't inherit from flutter app delegate? iirc they will have to copy some functionalities from flutter app delegate over.

Reply:
That seems to be covered in the next section, but the example is Objective-C. I'm not really sure what the Swift equivalent looks like. Could you provide some guidance?

Reply:
> That seems to be covered in the next section, but the example is Objective-C.

can you point me there? i can help to translate it into swift.

Reply:
Yes, the current section is here: https://docs.flutter.dev/development/add-to-app/ios/add-flutter-screen?tab=engine-objective-c-tab#using-the-flutterappdelegate

In the new version of the doc it's under a subheader ### If you can't directly make FlutterAppDelegate as subclass

Reply:
I translated this section to Swift here: 
https://github.com/hellohuanlin/put-flutter-to-work/commit/7515c37309cc839d24d969bf2f677bb897c49311

Also note that the existing objc section about the status bar seems outdated: https://github.com/flutter/flutter/issues/63051

So i did not translate that.

Reply:
@cyanglaz may know what happens to this status bar API.

Reply:
I don't think I have access to your version of put-flutter-to-work, can you grant me access?

Reply:
Done

Reply:
It was removed in https://github.com/flutter/engine/pull/16820, not sure about the details, but it looks like people don't need to use this API anymore. @gaaclarke to confirm.

Reply:
Cool thanks, I'm going to take it out for now.